core/vmpu/src/vmpu.c

```diff
@@ -27,6 +29,8 @@
 #include "vmpu_unpriv_access.h"
 #include <sys/reent.h>
 
+
+uint32_t g_debug_interrupt_sp[UVISOR_MAX_BOXES];
```

Comment:
Apparently it is. It has got there probably on the latest merge from Roman's branch...
We do not get any errors.
Should check why.